From 9ca2de8c5995657973665189903aa2eebe0ef69a Mon Sep 17 00:00:00 2001
From: David Ostrovsky <david@ostrovsky.org>
Date: Tue, 1 Dec 2015 00:20:09 +0100
Subject: Expose metadata to deploy LibreOffice artifacts on Maven Central

As outlined in the requirements to deploy the artifacts on Maven
Central, the metdata must be provided:

* Project Name, Description and URL
* License Information
* Developer Information
* SCM Information

[1] http://central.sonatype.org/pages/requirements.html

Change-Id: I0bcd19a22d0e1a48f0faec0b414f816f7da5b318
Reviewed-on: https://gerrit.libreoffice.org/20315
Tested-by: Jenkins <ci@libreoffice.org>
Reviewed-by: Thorsten Behrens <Thorsten.Behrens@CIB.de>

diff --git a/bean/pom.officebean.xml b/bean/pom.officebean.xml
new file mode 100644
index 0000000..8d366d2
--- /dev/null
+++ b/bean/pom.officebean.xml
@@ -0,0 +1,44 @@
+<project>
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>org.libreoffice</groupId>
+  <artifactId>officebean</artifactId>
+  <version>@version@</version>
+  <packaging>jar</packaging>
+  <name>LibreOffice - LOKit4Java</name>
+  <description>LOKit4Java</description>
+  <url>https://www.libreoffice.org</url>
+
+  <licenses>
+    <license>
+      <name>Mozilla Public License, Version 2.0</name>
+      <url>https://www.mozilla.org/en-US/MPL/2.0</url>
+      <distribution>repo</distribution>
+    </license>
+  </licenses>
+
+  <scm>
+    <url>http://cgit.freedesktop.org/libreoffice/core</url>
+    <connection>https://gerrit.libreoffice.org/#/admin/projects/core</connection>
+  </scm>
+
+  <developers>
+    <developer>
+      <name>The Document Foundation</name>
+    </developer>
+  </developers>
+
+  <mailingLists>
+    <mailingList>
+      <name>LibreOffice Development Mailing List</name>
+      <post>libreoffice@lists.freedesktop.org</post>
+      <subscribe>http://lists.freedesktop.org/mailman/listinfo/libreoffice</subscribe>
+      <unsubscribe>http://lists.freedesktop.org/mailman/listinfo/libreoffice</unsubscribe>
+      <archive>http://lists.freedesktop.org/archives/libreoffice</archive>
+    </mailingList>
+  </mailingLists>
+
+  <issueManagement>
+    <url>https://bugs.documentfoundation.org</url>
+    <system>LibreOffice Issue Tracker</system>
+  </issueManagement>
+</project>
diff --git a/javaunohelper/pom.juh.xml b/javaunohelper/pom.juh.xml
new file mode 100644
index 0000000..f132ead
--- /dev/null
+++ b/javaunohelper/pom.juh.xml
@@ -0,0 +1,44 @@
+<project>
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>org.libreoffice</groupId>
+  <artifactId>juh</artifactId>
+  <version>@version@</version>
+  <packaging>jar</packaging>
+  <name>LibreOffice - Java UNO helper</name>
+  <description>Java UNO helper</description>
+  <url>https://www.libreoffice.org</url>
+
+  <licenses>
+    <license>
+      <name>Mozilla Public License, Version 2.0</name>
+      <url>https://www.mozilla.org/en-US/MPL/2.0</url>
+      <distribution>repo</distribution>
+    </license>
+  </licenses>
+
+  <scm>
+    <url>http://cgit.freedesktop.org/libreoffice/core</url>
+    <connection>https://gerrit.libreoffice.org/#/admin/projects/core</connection>
+  </scm>
+
+  <developers>
+    <developer>
+      <name>The Document Foundation</name>
+    </developer>
+  </developers>
+
+  <mailingLists>
+    <mailingList>
+      <name>LibreOffice Development Mailing List</name>
+      <post>libreoffice@lists.freedesktop.org</post>
+      <subscribe>http://lists.freedesktop.org/mailman/listinfo/libreoffice</subscribe>
+      <unsubscribe>http://lists.freedesktop.org/mailman/listinfo/libreoffice</unsubscribe>
+      <archive>http://lists.freedesktop.org/archives/libreoffice</archive>
+    </mailingList>
+  </mailingLists>
+
+  <issueManagement>
+    <url>https://bugs.documentfoundation.org</url>
+    <system>LibreOffice Issue Tracker</system>
+  </issueManagement>
+</project>
diff --git a/jurt/pom.jurt.xml b/jurt/pom.jurt.xml
new file mode 100644
index 0000000..25e3403
--- /dev/null
+++ b/jurt/pom.jurt.xml
@@ -0,0 +1,44 @@
+<project>
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>org.libreoffice</groupId>
+  <artifactId>jurt</artifactId>
+  <version>@version@</version>
+  <packaging>jar</packaging>
+  <name>LibreOffice - Java Uno Runtime</name>
+  <description>Java Uno Runtime</description>
+  <url>https://www.libreoffice.org</url>
+
+  <licenses>
+    <license>
+      <name>Mozilla Public License, Version 2.0</name>
+      <url>https://www.mozilla.org/en-US/MPL/2.0</url>
+      <distribution>repo</distribution>
+    </license>
+  </licenses>
+
+  <scm>
+    <url>http://cgit.freedesktop.org/libreoffice/core</url>
+    <connection>https://gerrit.libreoffice.org/#/admin/projects/core</connection>
+  </scm>
+
+  <developers>
+    <developer>
+      <name>The Document Foundation</name>
+    </developer>
+  </developers>
+
+  <mailingLists>
+    <mailingList>
+      <name>LibreOffice Development Mailing List</name>
+      <post>libreoffice@lists.freedesktop.org</post>
+      <subscribe>http://lists.freedesktop.org/mailman/listinfo/libreoffice</subscribe>
+      <unsubscribe>http://lists.freedesktop.org/mailman/listinfo/libreoffice</unsubscribe>
+      <archive>http://lists.freedesktop.org/archives/libreoffice</archive>
+    </mailingList>
+  </mailingLists>
+
+  <issueManagement>
+    <url>https://bugs.documentfoundation.org</url>
+    <system>LibreOffice Issue Tracker</system>
+  </issueManagement>
+</project>
diff --git a/ridljar/pom.ridl.xml b/ridljar/pom.ridl.xml
new file mode 100644
index 0000000..96b862a0
--- /dev/null
+++ b/ridljar/pom.ridl.xml
@@ -0,0 +1,44 @@
+<project>
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>org.libreoffice</groupId>
+  <artifactId>ridl</artifactId>
+  <version>@version@</version>
+  <packaging>jar</packaging>
+  <name>LibreOffice - Types for the Java Uno typesystem</name>
+  <description>Types for the Java Uno typesystem</description>
+  <url>https://www.libreoffice.org</url>
+
+  <licenses>
+    <license>
+      <name>Mozilla Public License, Version 2.0</name>
+      <url>https://www.mozilla.org/en-US/MPL/2.0</url>
+      <distribution>repo</distribution>
+    </license>
+  </licenses>
+
+  <scm>
+    <url>http://cgit.freedesktop.org/libreoffice/core</url>
+    <connection>https://gerrit.libreoffice.org/#/admin/projects/core</connection>
+  </scm>
+
+  <developers>
+    <developer>
+      <name>The Document Foundation</name>
+    </developer>
+  </developers>
+
+  <mailingLists>
+    <mailingList>
+      <name>LibreOffice Development Mailing List</name>
+      <post>libreoffice@lists.freedesktop.org</post>
+      <subscribe>http://lists.freedesktop.org/mailman/listinfo/libreoffice</subscribe>
+      <unsubscribe>http://lists.freedesktop.org/mailman/listinfo/libreoffice</unsubscribe>
+      <archive>http://lists.freedesktop.org/archives/libreoffice</archive>
+    </mailingList>
+  </mailingLists>
+
+  <issueManagement>
+    <url>https://bugs.documentfoundation.org</url>
+    <system>LibreOffice Issue Tracker</system>
+  </issueManagement>
+</project>
diff --git a/ridljar/pom.unoloader.xml b/ridljar/pom.unoloader.xml
new file mode 100644
index 0000000..188a041
--- /dev/null
+++ b/ridljar/pom.unoloader.xml
@@ -0,0 +1,44 @@
+<project>
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>org.libreoffice</groupId>
+  <artifactId>unoloader</artifactId>
+  <version>@version@</version>
+  <packaging>jar</packaging>
+  <name>LibreOffice - UNO loader</name>
+  <description>UNO loader</description>
+  <url>https://www.libreoffice.org</url>
+
+  <licenses>
+    <license>
+      <name>Mozilla Public License, Version 2.0</name>
+      <url>https://www.mozilla.org/en-US/MPL/2.0</url>
+      <distribution>repo</distribution>
+    </license>
+  </licenses>
+
+  <scm>
+    <url>http://cgit.freedesktop.org/libreoffice/core</url>
+    <connection>https://gerrit.libreoffice.org/#/admin/projects/core</connection>
+  </scm>
+
+  <developers>
+    <developer>
+      <name>The Document Foundation</name>
+    </developer>
+  </developers>
+
+  <mailingLists>
+    <mailingList>
+      <name>LibreOffice Development Mailing List</name>
+      <post>libreoffice@lists.freedesktop.org</post>
+      <subscribe>http://lists.freedesktop.org/mailman/listinfo/libreoffice</subscribe>
+      <unsubscribe>http://lists.freedesktop.org/mailman/listinfo/libreoffice</unsubscribe>
+      <archive>http://lists.freedesktop.org/archives/libreoffice</archive>
+    </mailingList>
+  </mailingLists>
+
+  <issueManagement>
+    <url>https://bugs.documentfoundation.org</url>
+    <system>LibreOffice Issue Tracker</system>
+  </issueManagement>
+</project>
diff --git a/unoil/pom.unoil.xml b/unoil/pom.unoil.xml
new file mode 100644
index 0000000..356d25d
--- /dev/null
+++ b/unoil/pom.unoil.xml
@@ -0,0 +1,44 @@
+<project>
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>org.libreoffice</groupId>
+  <artifactId>unoil</artifactId>
+  <version>@version@</version>
+  <packaging>jar</packaging>
+  <name>LibreOffice - Maps IDL into java classes definitions</name>
+  <description>Maps IDL into java classes definitions</description>
+  <url>https://www.libreoffice.org</url>
+
+  <licenses>
+    <license>
+      <name>Mozilla Public License, Version 2.0</name>
+      <url>https://www.mozilla.org/en-US/MPL/2.0</url>
+      <distribution>repo</distribution>
+    </license>
+  </licenses>
+
+  <scm>
+    <url>http://cgit.freedesktop.org/libreoffice/core</url>
+    <connection>https://gerrit.libreoffice.org/#/admin/projects/core</connection>
+  </scm>
+
+  <developers>
+    <developer>
+      <name>The Document Foundation</name>
+    </developer>
+  </developers>
+
+  <mailingLists>
+    <mailingList>
+      <name>LibreOffice Development Mailing List</name>
+      <post>libreoffice@lists.freedesktop.org</post>
+      <subscribe>http://lists.freedesktop.org/mailman/listinfo/libreoffice</subscribe>
+      <unsubscribe>http://lists.freedesktop.org/mailman/listinfo/libreoffice</unsubscribe>
+      <archive>http://lists.freedesktop.org/archives/libreoffice</archive>
+    </mailingList>
+  </mailingLists>
+
+  <issueManagement>
+    <url>https://bugs.documentfoundation.org</url>
+    <system>LibreOffice Issue Tracker</system>
+  </issueManagement>
+</project>
-- 
cgit v0.10.2
